src/DotNet.Status.Web/Controllers/AzurePipelinesController.cs

```diff
@@ -237,15 +237,8 @@ private async Task ProcessBuildNotificationsAsync(Build build)
 
                     if (repo.UpdateExisting)
                     {
-                        // There is no way to get the username of our bot directly from the GithubApp with the C# api.
-                        // Issue opened in Octokit: https://github.com/octokit/octokit.net/issues/2335
-                        // We do, however, have access to the HtmlUrl, which ends with the name of the bot.
-                        // Additionally, when the bot opens issues, the username used ends with [bot], which isn't strictly
-                        // part of the name anywhere else. So, to get the correct creator name, get the HtmlUrl, grab
-                        // the bot's name from it, and append [bot] to that string.
-                        string creator = (await github.GitHubApps.GetCurrent()).HtmlUrl.Split("/").Last();
```

Comment:
This is not failing because "we can't use this". The failure is because its authenticated incorrectly. To use any of the methods under the "GitHubApps" path, the JWT encoded authentication is needed. The "github" here is authenticated using an installation token. This can 100% work, but it needs to be authed differently.

Reply:
we do this auth in here https://github.com/dotnet/arcade-services/blob/main/src/Shared/Microsoft.DotNet.GitHub.Authentication/GitHubTokenProvider.cs#L52

Reply:
Thank you! I will try that out

Reply:
Hm. It's definitely using that code. Not sure why it's not authing correctly.

Reply:
its using _gitHubApplicationClientFactory.CreateGitHubClientAsync, which uses that code I referenced to get an installation token, then authenticates with that token. That's the problem. The GitHubApps path is only usable with jwt authentication.

Reply:
More specifically the code is currently doing `auth with JWT -> acquire installation token -> auth with that token -> call .GitHubApps.GetCurrent()` it needs to do `auth with JWT -> call .GitHubApps.GetCurrent()`